spec/player_class_spec.rb

```diff
@@ -0,0 +1,103 @@
+require './lib/player_class.rb'
+require './lib/tilebag_class.rb'
+require './lib/scrabble_class.rb'
+require './lib/dictionary_class.rb'
+
+
+# minimum of 11 specs
+describe Scrabble::Player do
+
+    before :each do # (Before each thing I do, create a new player)
+      @bag = Scrabble::TileBag.new
+      @bag.draw_tiles(3)
+      @player1 = Scrabble::Player.new("Logan")
```

Comment:
It seems like you have a bunch of tests that just use `@player1` and then some that also just use `@bag`. I would recommend using the specific `before :each` with _only_ the instance variables that are required for those specific tests. This prevents your tests from taking longer by creating unnecessary variables.